app/routers/post.py

```diff
@@ -1,31 +1,80 @@
+from datetime import timedelta, datetime
 from .. import model, schemas, oauth2
 from fastapi import FastAPI, Response, status, HTTPException, Depends, APIRouter
 from sqlalchemy.orm import Session
 from typing import List, Optional
 from ..database import get_db
-from sqlalchemy import desc, func
+from sqlalchemy import desc, func, or_, and_
 
 router = APIRouter(prefix="/posts", tags=["Posts"])
 
 
-@router.get("/all", response_model=List[schemas.Post])
-def get_posts(
+@router.get("/all/story", response_model=List[schemas.Post])
+def get_all_story(
     db: Session = Depends(get_db), curr_user: int = Depends(oauth2.get_current_user)
 ):
-    posts = db.query(model.Post).all()
+    posts = (
+        db.query(model.Post)
+        .filter(
+            model.Post.expire > datetime.today(), model.Post.type_of_post == "story"
```

Comment:
We discussed about using enums here. I see it is changed in other PR so I'll merge this one